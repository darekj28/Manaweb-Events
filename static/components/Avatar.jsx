var React = require('react');

export default class Avatar extends React.Component {
	constructor(props) {
		super(props);
	}
	render() {
		return (
			<a href="#" className="pull-left AvatarSpace">
				<img className="Avatar img-rounded" src={this.props.source}/>
			</a>
			)
	}
}