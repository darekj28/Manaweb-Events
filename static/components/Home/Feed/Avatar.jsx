var React = require('react');

export default class Avatar extends React.Component {
	render() {
		return (
			this.props.source != undefined && <a href="#" className="pull-left AvatarSpace">
				<img className="Avatar" src={this.props.source}/>
			</a>
			)
	}
}