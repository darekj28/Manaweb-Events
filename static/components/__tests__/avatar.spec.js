import Avatar from '../Avatar.jsx';

describe('Avatar', () => {	
	it('renders correctly', () => {
		const src = "../../default.png";
		const avatar = renderer.create(
			<Avatar source={src}/>
			);
		expect(avatar).toMatchSnapshot();
	});
})