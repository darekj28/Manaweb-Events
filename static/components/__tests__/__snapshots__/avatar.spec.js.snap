exports[`Avatar renders correctly 1`] = `
<a
  className="pull-left AvatarSpace"
  href="#">
  <img
    className="Avatar img-rounded"
    src="../../default.png" />
</a>
`;
